fluent.conf

```diff
@@ -0,0 +1,48 @@
+##
+## Fluentd configurations
+## for Logz.io Logs
+## and Salesforce Commerce Cloud (aka Demandware)
+##
+<system>
+  <log>
+    time_format fluentd: %Y-%m-%d %H:%M:%S -
+  </log>
+</system>
+
+<source>
+  @type forward
+  @id input_forward
+</source>
+
+<label @FLUENT_LOG>
+  <match fluent.*>
+    @type null
+  </match>
+</label>
+
+
+<filter sfcc.*>
+  @type record_transformer
+  <record>
+    service_name ${tag}
+  </record>
+</filter>
+
+
+<match **>
+  @type logzio_buffered
+  endpoint_url "#{ENV['LOGZIO_LISTENER_URL']}/?token=#{ENV['LOGZIO_SHIPPING_KEY']}&type=cloud_commerce"
+
+  
+  output_include_time true
+  slow_flush_log_threshold "#{ENV['LOGZIO_SLOW_FLUSH_LOG_THRESHOLD']}"
```

Comment:
@resdenia You can set a default value in case this env is not used ( = null).
I'm not sure if this specific field requires an integer (5) or a string (something like 5s), so you'll need to change it to whatever is right, but it should look something like this:
`"#{ENV['LOGZIO_SLOW_FLUSH_LOG_THRESHOLD'] || 5 }"`
(You can see usage of this fallback value here: https://github.com/logzio/logzio-k8s/blob/master/configmap.yaml#L17)